db/sovereign-db/src/rocks_db_config.rs

```diff
@@ -0,0 +1,51 @@
+// Adapted from Aptos-Core.
+// Modified to remove serde dependency
+
+use rocksdb::Options;
+
+/// Port selected RocksDB options for tuning underlying rocksdb instance of AptosDB.
```

Comment:
Should we call it `SovDb` (last word of the above sentence.)

Reply:
Good catch!

Reply:
Fixed!